default.py

```diff
@@ -350,6 +357,43 @@ def select_version(self, game_versions):
         else:
             return None
 
+    def has_inputstream_adaptive(self):
+        """Checks if InputStream Adaptive is installed and enabled."""
+        payload = {
```

Comment:
I forgot Kodi had this JSONRPC interface. I'm assuming this is the only possible way to get this info? It feels so heavy-handed.